Sources/TuistCache/ContentHashing/ResourcesContentHasher.swift

```diff
@@ -19,7 +19,10 @@ public final class ResourcesContentHasher: ResourcesContentHashing {
     // MARK: - ResourcesContentHashing
 
     public func hash(resources: [FileElement]) throws -> String {
-        let hashes = try resources.map { try contentHasher.hash(path: $0.path) }
+        let hashes = try resources
```

Comment:
let me check if there are more arrays that aren't sorted 🙃  We already fixed a similar bug for another collection 😭